PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -255,19 +255,9 @@ void ScintillaEditView::init(HINSTANCE hInst, HWND hPere)
 	
 	_codepage = ::GetACP();
 
-	//Use either Unicode or ANSI setwindowlong, depending on environment

```

Comment:
Think this part is a leftover of the old days with ANSI and Unicode version (https://notepad-plus-plus.org/download/v5.9.6.2.html) , but 

_callWindowProc = CallWindowProcW;
should be
_callWindowProc = CallWindowProc;